### PR TITLE
Update PR template for autoclosing issues

### PR DESCRIPTION
**Pull request by peteward**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | If I had an issue to fix I would put it here with 'fixes #IT'
| License       | MIT

### CODE REVIEW

**Comment by adamelso** [comment]
Are there any circumstances where an issue may not necessarily be resolved by one pull request?

**Comment by peteward** [comment]
Almost certainly, I wouldn't add it to the `Fixes Ticket` section then if that was the case.

You could argue for another related tickets section or reword it like

 `Related Tickets  :  #X, fixes #Y`

**Comment by adamelso** [comment]
@peteward :+1:

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 | New feature?  | no|yes
 | BC breaks?    | no|yes
 | Deprecations? | no|yes
-| Fixed tickets | #X
+| Fixed tickets | fixes #X, fixes #Y
```
Comment (michalmarcinkowski) [inline PULL_REQUEST_TEMPLATE.md]: How about `fixes #X, fixes #Y, partially #Z`?

Comment (pamil) [inline PULL_REQUEST_TEMPLATE.md]: Related tickets | Fixes #X, mentioned in #Y, etc.?

Comment (peteward) [inline PULL_REQUEST_TEMPLATE.md]: `| Related tickets | fixes #X, partially #Y, mentioned in #Z ` ?

Comment (pjedrzejewski) [inline PULL_REQUEST_TEMPLATE.md]: :+1:

**Comment by peteward** [comment]
done!

**Comment by michalmarcinkowski** [comment]
Great! Thanks Peter :+1:

**Pull request merged**